platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/internal/component/resolution/failure/ResolutionFailureHandlerIntegrationTest.groovy

```diff
@@ -150,6 +151,14 @@ class ResolutionFailureHandlerIntegrationTest extends AbstractIntegrationSpec {
             additionalData.asMap['problemId'] == ResolutionFailureProblemId.AMBIGUOUS_VARIANTS.name()
             additionalData.asMap['problemDisplayName'] == "Multiple variants exist that would match the request"
         }
+
+        if (GradleContextualExecuter.configCache) {
+            verifyAll(receivedProblem(1)) {
+                fqid == 'validation:configuration-cache:error-writing-value-of-type-org-gradle-api-internal-file-collections-defaultconfigurablefilecollection'
+                contextualLabel == 'error writing value of type \'org.gradle.api.internal.file.collections.DefaultConfigurableFileCollection\''
+                additionalData.asMap['trace'] == 'task `:forceResolution` of type `Build_gradle$ForceResolution`'
+            }
+        }
```

Comment:
🤔 Instead of anticipating the errors here, it would be better to update the test scenarios to not produce them.

Reply:
OK.  I believe this test _was_ fully CC compatible at some point - it's very possible something else may have changed.  If you didn't break it though, I suppose it's not your job to fix it. 😄   

I'll take the issue and update these soon.